tests/testthat/test.py

```diff
@@ -0,0 +1,21 @@
+import MACS2
+import argparse
+from MACS2 import filterdup_cmd
+from MACS2 import OptValidator
+
+class Namespace:
+    def __init__(self, **kwargs):
+        self.__dict__.update(kwargs)
+
+options = Namespace(gsize = 2.7e9,
+                    format = "BEDPE", 
+                    keepduplicates = "1",
+                    verbose = 1,
+                    outputfile = "CTCF_ChIP_200K_filterdup.bed",
+                    outdir = "/tmp",
+                    ifile = ["/Users/qi28068/Workspace/MACSr/tests/testthat/CTCF_PE_CTRL_chr22_50k.bedpe.gz"],
```

Comment:
Let's include the testing files in the package. You can use the files under https://github.com/macs3-project/MACS/tree/master/test

Reply:
Will add test in the next pull